test/utils/time.js

```diff
@@ -0,0 +1,16 @@
+const { mineNextBlock, increaseTime: increaseTimeBy, setNextBlockTime } = require('./evm');
+
+const setTime = async timestamp => {
+  await setNextBlockTime(timestamp);
+  await mineNextBlock();
+};
+
+const increaseTime = async timeIncrease => {
+  await increaseTimeBy(timeIncrease);
+  await mineNextBlock();
+};
+
+module.exports = {
+  increaseTime,
+  setTime,
+};
```

Comment:
Please use `setNextBlockTime` and `increaseTimeBy` directly in the tests. An important argument for not having this functions is that they mine a block. If as a developer you'd want at some point to mine a tx at a specific time, `setTime` usage before the tx will make the tx get mined one second later due to the `mineNextBlock` in this function. The current existing functions are sufficient and are explicit - `setNextBlockTime` function name tells the developer exactly what will happen and he'll be able to decide if he needs to use `mineNextBlock` or not.